spec/controllers/boards_controller_spec.rb

```diff
@@ -0,0 +1,159 @@
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator.  If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails.  There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+#
+# Compared to earlier versions of this generator, there is very limited use of
+# stubs and message expectations in this spec.  Stubs are only used when there
+# is no simpler way to get a handle on the object needed for the example.
+# Message expectations are only used when there is no simpler way to specify
+# that an instance is receiving a specific message.
+
+RSpec.describe BoardsController, type: :controller do
+
+  # This should return the minimal set of attributes required to create a valid
+  # Board. As you add validations to Board, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
+  }
+
+  let(:invalid_attributes) {
+    skip("Add a hash of attributes invalid for your model")
+  }
+
+  # This should return the minimal set of values that should be in the session
+  # in order to pass any filters (e.g. authentication) defined in
+  # BoardsController. Be sure to keep this updated too.
+  let(:valid_session) { {} }
+
+  describe "GET #index" do
+    it "assigns all boards as @boards" do
+      board = Board.create! valid_attributes
+      get :index, params: {}, session: valid_session
+      expect(assigns(:boards)).to eq([board])
+    end
+  end
+
+  describe "GET #show" do
+    it "assigns the requested board as @board" do
+      board = Board.create! valid_attributes
+      get :show, params: {id: board.to_param}, session: valid_session
+      expect(assigns(:board)).to eq(board)
+    end
+  end
+
+  describe "GET #new" do
+    it "assigns a new board as @board" do
+      get :new, params: {}, session: valid_session
+      expect(assigns(:board)).to be_a_new(Board)
+    end
+  end
+
+  describe "GET #edit" do
+    it "assigns the requested board as @board" do
+      board = Board.create! valid_attributes
+      get :edit, params: {id: board.to_param}, session: valid_session
+      expect(assigns(:board)).to eq(board)
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid params" do
+      it "creates a new Board" do
+        expect {
+          post :create, params: {board: valid_attributes}, session: valid_session
+        }.to change(Board, :count).by(1)
+      end
+
+      it "assigns a newly created board as @board" do
+        post :create, params: {board: valid_attributes}, session: valid_session
+        expect(assigns(:board)).to be_a(Board)
+        expect(assigns(:board)).to be_persisted
```

Comment:
wazne jest by pisac testy z jednym expect w `'it` bo inaczej ciezko opisac taki test